### PR TITLE
6160: HTML JSP Warnings

### PR DESCRIPTION
**Pull request by RosalineP**

### CODE REVIEW

File: flow/src/org/labkey/flow/controllers/executescript/importAnalysisReviewSamples.jsp [hunk 1]
```diff
@@ -35,6 +35,7 @@
 <%@ page import="java.util.List" %>
 <%@ page import="java.util.Map" %>
 <%@ page import="java.util.TreeMap" %>
+<%@ page import="org.labkey.api.util.HtmlString" %>
```
Comment (labkey-adam) [inline flow/src/org/labkey/flow/controllers/executescript/importAnalysisReviewSamples.jsp]: Remove unused import?

File: luminex/src/org/labkey/luminex/view/guideSetConfirmDelete.jsp [hunk 2]
```diff
@@ -60,7 +61,7 @@
             <br>
             Type: <% if(gs.isValueBased()) out.print("Value-based"); else out.print("Run-based"); %>
             <br><br>
-            Current Guide Set: <%=gs.getCurrent()%>
+            Current Guide Set: <%=HtmlString.of(gs.getCurrent().toString())%>
```
Comment (labkey-adam) [inline luminex/src/org/labkey/luminex/view/guideSetConfirmDelete.jsp]: I would just call h() on Booleans. It's simpler... plus it handles the case where gs.getCurrent() is null. (This code will NPE.)

Comment (labkey-adam) [reply luminex/src/org/labkey/luminex/view/guideSetConfirmDelete.jsp]: We could consider adding an h(Boolean) variant to JspBase, but I think the h(Object) variant is essentially the same thing.

Comment (RosalineP) [reply luminex/src/org/labkey/luminex/view/guideSetConfirmDelete.jsp]: Good point!

File: flow/src/org/labkey/flow/controllers/executescript/importAnalysisReviewSamples.jsp (hunk 1 above)
Comment (RosalineP) [reply flow/src/org/labkey/flow/controllers/executescript/importAnalysisReviewSamples.jsp]: Oops, thanks for the catch!

File: luminex/src/org/labkey/luminex/view/guideSetConfirmDelete.jsp [hunk 3]
```diff
@@ -23,6 +23,7 @@
 <%@ page import="org.labkey.luminex.LuminexController" %>
 <%@ page import="java.util.List" %>
 <%@ page import="org.labkey.api.view.ActionURL" %>
+<%@ page import="org.labkey.api.util.HtmlString" %>
```
Comment (labkey-adam) [inline luminex/src/org/labkey/luminex/view/guideSetConfirmDelete.jsp]: Now this is an unused import :) But let's not worry about it...

**Pull request merged**